staging/src/k8s.io/cli-runtime/pkg/resource/builder_test.go

```diff
@@ -1051,7 +1052,10 @@ func TestRestMappingErrors(t *testing.T) {
 	err := b.Do().IntoSingleItemImplied(&singleItemImplied).Visit(test.Handle)
 	if err != nil {
 		if !strings.Contains(err.Error(), "server doesn't have a resource type \"foo\"") {
-			t.Fatalf("unexpected error: %v", err)
+			t.Fatalf("unexpected error message: %v", err)
```

Comment:
Why did you change the current error message?

Reply:
Reverted.